integrationtest/v5migrationhelper/orm/src/test/java/org/hibernate/search/test/batchindexing/DatabaseMultitenancyTest.java

```diff
@@ -43,10 +43,6 @@
  * @author Sanne Grinovero
  * @since 5.2
  */
-@RequiresDialect(
```

Comment:
Note that I've removed any `RequiresDialect` or `SkipForDialect` ... 
They do not work OK with testcontainers... the DialectContext in the ORM test utils wants to have connection properties in env and it even requires a Dialect variable too.... so it looks like we can just leverage the dialect from the configuration of the container and do not rely on this util.

Reply:
Ok... does this mean we (well, more likely QE) can no longer reliably test Hibernate Search against non-container databases?

Reply:
yeah ... 
there was this https://github.com/hibernate/hibernate-search/pull/3796#discussion_r1368828689 that was suppose to be something similar to ES config, if someone somehow provides a way to connect to the DB then we woulnd't start a container... but I figured you were saying that I should remove it 😄 

now if we want to be able to test against non-testcontainer DB... we can do that in the `DatabaseContainer#configuration()`. Just let me know which property we'd expect a user to pass to provide connection info (will `jakarta.persistence.jdbc.url` be ok?)

Reply:
> but I figured you were saying that I should remove it 😄

I think my main problem was with getting the info from a system property...

> Just let me know which property we'd expect a user to pass to provide connection info (will jakarta.persistence.jdbc.url be ok?)

I'd expect people to do it more or less the same as today, so:

* For Elasticsearch/OpenSearch, I think it's documented in `CONTRIBUTING.md`
* For databases, I think currently you'd have to pass something like `-Pci-postgresql -Dtest.database.run.skip=true -Djdbc.url=...` and optionally (if using different settings than what Hibernate Search expects `-Djdbc.user=... -Djdbc.pass=...`.

Reply:
> For databases, I think currently you'd have to pass something like `-Pci-postgresql -Dtest.database.run.skip=true -Djdbc.url=...` and optionally (if using different settings than what Hibernate Search expects `-Djdbc.user=... -Djdbc.pass=...`.

But feel free to align this on whatever you'll do for Elasticsearch/OpenSearch

Reply:
ok 😃 
`test.database.run.skip=true` -- we wouldn't need this one anymore (same as it is for the Elasticsearch case)
`-Djdbc.url` -- will make this work 👍🏻 

In general, the idea is that the `*Container` checks whether the connection URL was provided, and only if not then it starts a container, otherwise -- it is using the provided URL.